Assets/Scripts/VrSdk.cs

```diff
@@ -12,10 +12,6 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#if OCULUS_SUPPORTED || ZAPBOX_SUPPORTED
```

Comment:
what happened to this?

Reply:
I don't think this section was ever needed; it enabled `PASSTHROUGH_SUPPORTED` inside `OCULUS_SUPPORTED`! Andy removed the section in #715

Reply:
Good point! I missed that (that's what happens when you review in Github without looking at the full tree)